pages/windows/pipwin.md

```diff
@@ -0,0 +1,28 @@
+# pipwin
+
+> Complementary tool to install unofficial Python package binaries on Windows.
+> More information: <https://github.com/lepisma/pipwin>.
+
+- List all available packages for download:
+
+`pipwin list`
+
+- Search packages:
+
+`pipwin search {{partial_name/name}}`
```

Comment:
```suggestion
`pipwin search {{partial_name|name}}`
```

What is a partial name, could you provide an example of both.

Reply:
yes, sure.
Due to information provided in developer github page, I've used "partial name" :
 
 `pipwin search plot`
 
 Did you mean any of these ?

 * bqplot
 * ggplot
 * hvplot
 * matplotlib
 * matplotlib-inline
 * plotly
 * plotnine
 * scikits.vectorplot